app/data/schema/schema-v1.json

```diff
@@ -56,6 +59,15 @@
     },
     "groups": {
       "type": "array",
+      "completed_id": {
+          "type":"string"
+      },
+      "highlight_when": {
+          "type":"array"
+      },
+      "hide_in_navigation": {
+          "type":"boolean"
```

Comment:
We should start adding "description": "This is what a hide_in_navigation type is"